### DISCUSSION TITLE
Should we release 0.68.2?

### DISCUSSION THREAD

**Discussion by cortinico** [reactwg/react-native-releases#20] [Patches]
Current Release: [0.68.1](https://github.com/facebook/react-native/releases/tag/v0.68.1)
    
Conversations on this thread are limited: 
* [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking)
* [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147
    
In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue. 
If the issue is a [major release issue](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.
    
---
#### List of qualified picks
 
1. https://github.com/facebook/react-native/commit/62ef6f5fa1ecb918bde130a6024b65afcd34c7e3 

#### Local commits to backport to main
1.

**Comment by joe-sam** [comment]
The configuration option should in fact override the default but doesn't as described in issue [32239](https://github.com/facebook/react-native/issues/32239).
 This appears to be an _android only + hermes specific_ related bug for generating the intermediate packager sourcemap from configuration , which is apparently still present in [react.gradle](https://github.com/facebook/react-native/blob/v0.68.1/react.gradle#L184). 
 
 Unfortunately  there is no related commit but the patch included does describes a simple one line fix.

**Reply by fortmarek** [reply]
Hey @joe-sam!

Can you create a new issue with reproducible steps with the latest version? We can continue the discussion there. Unfortunately, there's nothing we can cherry-pick until it gets fixed in `main`.

**Reply by joe-sam** [reply]
Sorry @fortmarek, thanks for your response. I understand that due process needs to be followed. 
After examining and testing the patch I decided that it was not actually a one-line fix after all, therefore I have submitted a  [PR #33703](https://github.com/facebook/react-native/pull/33703) to resolve the issue, and to preserve the existing usual workflow behaviour also in order to avoid any breaking change, and we shall wait till it gets fixed in main before cherry-picking.

**Comment by pedrolopes10** [comment]
This one is merged in master:
https://github.com/facebook/react-native/commit/c231d5e371cbddf0e54bf84f46336b22bbdb5367

**Reply by fortmarek** [reply]
Hey @pedrolopes10 👋 

Thanks for the suggestion, however, this seems to be an optimization and not something that fixes a bug introduced in `0.68`. I think we should wait with this for 0.69 unless this is somehow urgent (re: [qualified picks](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request))

**Comment by brunoducattii** [comment]
Hope that 0.68.2 will bring upgrades about enabling the New Architecture. This certainly will be a milestone for countries like Brazil where the average people has medium quality Androids. There are several new issues (even with iOS) about enabling it and app crashing: https://github.com/facebook/react-native/issues?page=2&q=is%3Aissue+is%3Aopen
Our company tried to handle with that for several weeks but because we are a lean startup we can't help putting more efforts to help the community. Thanks to everyone working on it!

Good luck.

**Reply by cortinico** [reply]
Thanks for your insigths @joaotmachado. We value user feedback as yours while we rollout the new architecture, and we're working hard to address them.

**Comment by kelset** [comment]
posting it here in order not to miss it - the PR that should properly fix the Windows + newArch issue (https://github.com/facebook/react-native/issues/33528) is here: https://github.com/facebook/react-native/pull/33707

**Comment by mateusz1913** [comment]
https://github.com/facebook/react-native/commit/709a459b1ef7968d447c0a5dbe5cd44d1e03fcb4 this fixes pod install errors for 3rd party "new architecture"-enabled with Swift code

**Reply by fortmarek** [reply]
Thanks for the suggestion, we have decided to cherry-pick this for 0.69.0 only instead (expect it in the next RC release)
